R/class2tree.R

```diff
@@ -56,18 +56,26 @@ class2tree <- function(input, varstep = TRUE, check = TRUE, ...) {
     message('Removed species without classification.')
     input <- input[!is.na(input)]
   }
+
   # Check that there is more than 2 taxon
   if (length(input) < 3)
     stop("Your input list of classifications must be 3 or longer.")
+
   dat <- rbind.fill(lapply(input, class2tree_helper))
-  df <- dat[ , !apply(dat, 2, function(x) any(is.na(x))) ]
-  if (!inherits(df, "data.frame")) {
```

Comment:
fine to remove this, but is there now no chance of resulting in no ranks in common?

Reply:
uhm, I don't think so. Since we are taking into account even [the root rank](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=1), so 2 taxa will always have at least one rank in common. Therefore I think checking for ```!inherits(df, "data.frame")```not necessary.

Reply:
ok, thanks